### DISCUSSION TITLE
Ref of element is undefined when using nested object

### DISCUSSION THREAD

**Discussion by 16TheHung10** [react-hook-form/react-hook-form#10952] [General]
This is my Component with name "sourceItem[1].itemCode": and element Input is imported from antd;
<Controller
                    control={control}
                    name={name}
                    render={({ field }) => {
                      return (
                        <Input
                          className={`${get(formState.errors, name) ? 'field-error' : ''}`}
                          {...field}
                          {...props}
                          type={type}
                          onBlur={(e) => {
                            props.onBlur && props.onBlur(e);
                            field.onBlur(e);
                          }}
                        />
                      );
                    }}
                  />
When I console.log errors from formState, the ref to element is undefined , but when Im using normal name like "itemCode" everything is ok.

**Comment by leapful** [comment]
@16TheHung10 

Please provide a codesandbox url to reproduce the issue.
